### PR TITLE
import/export

### PR DESCRIPTION
**Pull request by metmarkosaric**

### CODE REVIEW

File: docs/google-analytics-import.md [hunk 1]
```diff
@@ -4,105 +4,105 @@ title: Import stats from Google Analytics
...
-:::
+4. On the next screen, you can click on the "**Confirm import**" button to start the data import process. Depending on the amount of data you have, this may take some time. You'll receive an email when the import has been completed. It should take no more than a couple of minutes for most sites.
 
-4. On the next screen, you can click on the "**Confirm import**" button to start the data import process. Depending on the amount of data you have, this may take some time. This is run in the background, so you will receive an email when the import has been completed. It should take no more than a couple of minutes for most sites.
+5. Once you have received the email, the imported data will be visible in your Plausible dashboard alongside the native data. An icon is shown in the top-right of the dashboard when imported data has been included. Clicking this icon will stop the imported data from being displayed and the icon will then appear with a strike-through.
```
Comment (RobertJoonas) [inline docs/google-analytics-import.md:23]:
```suggestion
5. Once you have received the email, the imported data will be visible in your Plausible dashboard alongside the native data. An icon is shown in the top-right of the dashboard when imported data has been included. Clicking this icon will stop the imported data from being displayed and the icon will then appear with a faded color.
```

**Review by RobertJoonas — COMMENTED** [review]
@metmarkosaric I think we want to merge some of this, but keep out the CSV export part at this stage.

**Comment by metmarkosaric** [comment]
@RobertJoonas yeah basically changes to google-analytics-import.md and your-plausible-experience.md are related to the GA4 import and should be merged only. the rest is related to the CSV export/import so that one can stay as a PR until that work is ready

**Comment by metmarkosaric** [comment]
this one is not needed any more with https://github.com/plausible/docs/pull/493

**Pull request closed without merging**